### PR TITLE
Asus: Remove System.Management

### PR DESCRIPTION
**Pull request by Aytackydln**
I've had dependency problem with Aurora and my only solution was this. I know this is project specific change but I think this dependency is unnecessary anyway.

It should be removed so that it won't be problem for other projects in the future.

### CODE REVIEW

**Comment by DarthAffe** [comment]
what is the issue with the dependency?
since this is quite a major breaking change for the provider i'd prefer to fix the actual problem.

**Comment by Aytackydln** [comment]
Error was like "Interop.AuraServiceLib.dll not found" or something. Research on internet said it was version conflict. I used various tools to figure out the conflict but none of them gave any clue. Difference between Aurora and Artemis was .net version and System.* dependencies. Removing System.Management from this library fixed my problem

**Comment by DarthAffe** [comment]
This doesn't make any sense - if the error is "Interop.AuraServiceLib.dll not found" then it normally is exactly this - sometimes aura does not correctly register it in the GAC, that's why artemis is distributing it.

**Comment by Aytackydln** [comment]
I made sure to put the dll everywhere it can be found...

**Comment by DarthAffe** [comment]
yeah, but this change is also breaking device detection for some devices and i really can't see any relation between that dependency and the capability of finding a library.
can you provide a full error message and stacktrace for the error?

**Comment by DarthAffe** [comment]
from looking at it i'd say you have your version conflict in aurora as you're explicitely referencing 6.0.0 of System.Management, the provider uses 7.0.0.
i still don't see why that would cause in issue related to the interop lib but you'd need to generate binding redirects to fix that (or update the lib in aurora)

**Comment by Aytackydln** [comment]
Well I'm out of arguments :)

**Comment by MesrineNathanael** [comment]
Thank you for your fix, it fixed the issue with the asus aura devices for me too, I was having the same issue as you had :)

File: RGB.NET.Devices.Asus/Keyboard/AsusKeyboardRGBDeviceInfo.cs [hunk 1]
```diff
@@ -37,7 +36,7 @@ internal AsusKeyboardRGBDeviceInfo(IAuraSyncDevice device)
 
     #region Methods
 
-    private static string? GetKeyboardModel(string deviceName) => GENERIC_DEVICE_NAMES.Contains(deviceName) ? WMIHelper.GetSystemModelInfo() : deviceName;
+    private static string? GetKeyboardModel(string deviceName) => GENERIC_DEVICE_NAMES.Contains(deviceName) ? "Asus Motherboard" : deviceName;
```
Comment (DarthAffe) [inline RGB.NET.Devices.Asus/Keyboard/AsusKeyboardRGBDeviceInfo.cs:39]:
```suggestion
    private static string? GetKeyboardModel(string deviceName) => GENERIC_DEVICE_NAMES.Contains(deviceName) ? "Asus Keyboard" : deviceName;
```

**Review by DarthAffe — CHANGES_REQUESTED** [review]

**Review by DarthAffe — APPROVED** [review]

**Pull request merged**